code/modules/shuttle/marine_dropship.dm

```diff
@@ -539,7 +545,7 @@
 	var/obj/docking_port/mobile/marine_dropship/M = SSshuttle.getShuttle(shuttleId)
 	if(!M)
 		return
-	if(M.hijack_state != HIJACK_STATE_NORMAL)
+	if(M.hijack_state == HIJACK_STATE_CALLED_DOWN)
 		return
```

Comment:
Why?

Reply:
we want the pilot to be able to use the consol in cockpit, else the hive leader unlocking the alamo would be a sure death sentence for marines. Here they have a chance to get out